src/DssSpell.sol

```diff
@@ -30,10 +25,6 @@ contract SpellAction {
     // The contracts in this list should correspond to MCD core contracts, verify
     //  against the current release list at:
     //     https://changelog.makerdao.com/releases/mainnet/active/contracts.json
-    ChainlogAbstract constant CHANGELOG = ChainlogAbstract(0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F);
```

Comment:
We might want to leave the `CHANGELOG ` this in the spell as it is always used.

Reply:
Added this back.